op-e2e/op_geth_test.go

```diff
@@ -90,6 +90,7 @@ func TestPreregolith(t *testing.T) {
 	for _, test := range tests {
 		test := test
 		t.Run("GasUsed_"+test.name, func(t *testing.T) {
+			InitParallel(t)
```

Comment:
Adding parallelism to subtests is a huge improvement, very nice!

Reply:
This actually doesn't make much of a difference when running the entire test suite. Parallelizing these tests takes it from 2.5s -> 1s, however the top level tests are already running in parallel with the other top level tests & the total test time even with maximally parallel tests is order 3-4 minutes.

Reply:
Yeah its a minor thing, but I thought just the top level call was enough to run them all fully parallel and it turns out its not so... As much as anything its just setting up the pattern that you always call `InitParallel` as the first thing in the test, then if we need to do any common setup like setting up the geth logging we can do it in that one method, plus all tests are built running in parallel from the start to do what we can to keep build times down.